### DISCUSSION TITLE
Publish AppImage, Flatpack, and Debian / Ubutnu packages

### DISCUSSION THREAD

**Discussion by yonas** [ghostty-org/ghostty#3593] [Feature Requests, Ideas] [answered]
Would be more convenient that building from source.

**Comment by mitchellh** [comment, marked as answer]
The Ghostty project doesn't officially maintain any packages. Please feel free to be the change you want in the world: https://github.com/ghostty-org/ghostty/blob/main/PACKAGING.md

**Reply by KaKi87** [reply]
Why are you doing it for Mac but not for Linux ?

**Reply by mitchellh** [reply]
Mac requires code signing and notarization which costs money, and it's not typical even in FOSS for app bundles to be maintained by the community. For Linux, it is typical.

**Reply by KaKi87** [reply]
From my experience, most of the time, developers provide manually downloadable DEBs or AppImage even if they don't provide automatically updated APT repo or Flatpak.

Anyway, popular desktop environments already provide great default terminal emulators and popular independent ones packaged by maintainers exist too.

So, yours is probably gonna remain a Mac-only one for a while... (Arch-based distros are a negligible market share).

**Reply by mitchellh** [reply]
That’s fine with me! I believe the distros will get it in time :) I’m just happy to share my work for those who are interested. If not, also great!
